pennylane_lightning/core/src/gates/Constant.hpp

```diff
@@ -94,6 +94,54 @@ namespace Pennylane::Gates::Constant {
         GateOperation::DoubleExcitationPlus, "DoubleExcitationPlus"},
     std::pair<GateOperation, std::string_view>{GateOperation::MultiRZ,
                                                "MultiRZ"}};
+
```

Comment:
You may be able to make this block of code more compact with:
```suggestion
    typedef std::pair<ControlledGateOperation, std::string_view> CGateView;
```
This name is only a suggestion.